### DISCUSSION TITLE
How to solve this problem

### DISCUSSION THREAD

**Discussion by wolfccc15** [Netflix/OpenVPCal#57] [General]
An unhandled Exception Occurred
list index out of range

![微信截图_20240815110703](https://github.com/user-attachments/assets/81053599-cf23-4f9c-9b6e-0bfc417b40e2)

I had this problem after importing photos

**Comment by Federico-Catelotti** [comment]
I have the same problem. Have you find a solution for that?

**Comment by giardiello** [comment]
Hello,Have you tried the latest version released before Christmas?On 8 Jan 2025 20:24, Federico Catelotti ***@***.***> wrote:
I have the same problem. Have you find a solution for that?
